package.json

```diff
@@ -5,32 +5,34 @@
   "license": "MIT",
   "repository": "danielhusar/gulp-stylint",
   "author": {
-    "name": "Daniel Husar",
     "email": "dano.husar@gmail.com",
+    "name": "Daniel Husar",
```

Comment:
All changes here, except for adding `jshint`, were made by npm itself when I saved the dep. I think it makes stuff alphabetical